### ISSUE TITLE
end of central directory record signature not found

### ISSUE THREAD

**Issue by chrisolsen** [Microsoft/vscode#28621]
- VSCode Version: 1.13.0
- OS Version: Windows 10

Steps to Reproduce:

1. Try to update to the latest version of vscodevim (this is the only plugin I have issues with)

I am not running behind any proxy (as far as I know). I have the issue when I'm at home or work on the same laptop. I do however use Fiddler during development, so it's possible the proxy it sets up is not being removed, but I'm not sure how to determine if this is true or not.

In a last ditch attempt I tried uninstalling the plugin, then reinstalling, now I have no plugin and crippled vim-dependent typing skills.

This may be a vscodevim issue, if so let me know.

**Comment by ramya-rao-a** [comment]
Are there any errors that pop up?
Any errors in the console? You can open the console by running the command `Developer: Toggle Developer Tools`.

**Comment by chrisolsen** [comment]
Here is the error and the stack trace.
```
end of central directory record signature not found: Error: end of central directory record signature not found,    at C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\yauzl\index.js:98:14,    at C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\yauzl\index.js:342:5,    at C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\fd-slicer\index.js:32:7,    at FSReqWrap.wrapper [as oncomplete] (fs.js:603:17)

	e.doShow	@	messageService.ts:125
e.show	@	messageService.ts:104
(anonymous)	@	extensionsList.ts:83
r	@	eventEmitter.ts:295
e._emitToSpecificTypeListeners	@	eventEmitter.ts:162
e._emitEvents	@	eventEmitter.ts:181
(anonymous)	@	eventEmitter.ts:129
r	@	eventEmitter.ts:295
e._emitToBulkListeners	@	eventEmitter.ts:170
e._emitEvents	@	eventEmitter.ts:176
e.emit	@	eventEmitter.ts:196
(anonymous)	@	actions.ts:232
v	@	winjs.base.raw.js:1209
enter	@	winjs.base.raw.js:901
_run	@	winjs.base.raw.js:1068
_chainedError	@	winjs.base.raw.js:1031
n	@	winjs.base.raw.js:736
v	@	winjs.base.raw.js:1209
enter	@	winjs.base.raw.js:901
_run	@	winjs.base.raw.js:1068
_error	@	winjs.base.raw.js:1041
e.doRequest.n.Promise.t.handlers.(anonymous function)	@	ipc.ts:254
e.onMessage	@	ipc.ts:319
(anonymous)	@	ipc.ts:206
e.invoke	@	callbackList.ts:63
e.fire	@	event.ts:123
(anonymous)	@	ipc.net.ts:82
emitOne	@	events.js:96
emit	@	events.js:188
readableAddChunk	@	_stream_readable.js:176
Readable.push	@	_stream_readable.js:134
onread
```

**Comment by Anthropic** [comment]
I'm getting the same issue with the `sleistner.vscode-fileutils` extension so I assume it would be any that need updating. I'm on VSCode Version: 1.13.0 - Win7 sp1

I'm getting the same log entry in my debugger console as above.

**Comment by Anthropic** [comment]
![image](https://user-images.githubusercontent.com/1182507/27109078-0232ee46-50e4-11e7-9275-0a35100e82d8.png)

**Comment by joaomoreno** [comment]
@Anthropic Are you behind a proxy?

@chrisolsen How do you configure Code to use Fiddler?

**Comment by chrisolsen** [comment]
No configuration is required with Fiddler.

**Comment by imba-tjd** [comment]
I have the same issue with `ms-vscode.cpptools` extension.
* VS Code version:1.12-1.13.1
* OS Version:Windows 10 LTSB 10240(latest upgrade)

I didn't use Fiddler.
I had installed Apache 24 a few weeks before, but cpptools didn't had an update at that time and I haven't used VS Code to write C code for a while. So I don't know whether there is a connection between them. And I have uninstalled Apache but it didn't work.
I do use Lantern VPN sometimes but I think it can exit clearly.
<img width="555" alt="367" src="https://user-images.githubusercontent.com/24759802/27176929-2a03d336-51f7-11e7-9f8a-57569527bcb5.png">
